docs/guide/add.md

```diff
@@ -96,23 +96,55 @@ gpio add kdtree input.parquet output.parquet --verbose
 
 ## Administrative Divisions
 
-Add country ISO codes via spatial join:
+Add administrative division columns via spatial join with remote boundaries datasets:
+
+### How It Works
+
+Performs spatial intersection between your data and remote admin boundaries to add admin division columns. Uses efficient spatial extent filtering to query only relevant boundaries from remote datasets.
+
+### Quick Start
+
+```bash
+# Add all GAUL levels (continent, country, department)
+gpio add admin-divisions input.parquet output.parquet --dataset gaul
+
+# Preview SQL before execution
+gpio add admin-divisions input.parquet output.parquet --dataset gaul --dry-run
+```
+
+### Multi-Level Admin Divisions
+
+Add multiple hierarchical administrative levels:
 
 ```bash
-# Use default countries dataset
-gpio add admin-divisions buildings.parquet output.parquet
+# Add all GAUL levels (adds admin:continent, admin:country, admin:department)
+gpio add admin-divisions buildings.parquet output.parquet --dataset gaul
 
-# Use custom countries file
-gpio add admin-divisions buildings.parquet output.parquet \
-  --countries-file my_countries.parquet
+# Add specific levels only
+gpio add admin-divisions buildings.parquet output.parquet --dataset gaul \
+  --levels continent,country
 
-# Preview SQL
-gpio add admin-divisions buildings.parquet output.parquet --dry-run
+# Use Overture Maps dataset
+gpio add admin-divisions buildings.parquet output.parquet --dataset overture \
+  --levels country,region
 ```
 
-Uses the [administrative division extension](https://github.com/fiboa/administrative-division-extension) from [fiboa](https://github.com/fiboa).
+### Datasets
+
+Two remote admin boundary datasets are supported:
+
+| Dataset | Columns Added | Description |
+|---------|--------------|-------------|
+| `gaul` (default) | `admin:continent`, `admin:country`, `admin:department` | GAUL L2 Admin Boundaries - worldwide |
+| `overture` | `admin:country`, `admin:region` | Overture Maps Divisions (219 countries, 3,544 regions) - [docs](https://docs.overturemaps.org/guides/divisions/) |
+
+### Notes
 
-Default countries source: [source.coop admin boundaries](https://data.source.coop/cholmes/admin-boundaries/countries.parquet)
+- Uses the [administrative division extension](https://github.com/fiboa/administrative-division-extension) from [fiboa](https://github.com/fiboa)
+- GAUL dataset: [source.coop GAUL L2](https://data.source.coop/nlebovits/gaul-l2-admin/)
+- Performs spatial intersection to assign admin divisions based on geometry
+- Requires internet connection to access remote datasets
```

Comment:
Is this now required? My original implementation just used a local one. I like that it can now work without a user having to figure that out, but I think it'd be good to prefer local, as it'll generally be faster.

One idea is that we can have a default cache location for the admin files (let users set it differently if they want, and provide an argument or environment variable). So it'd just fetch it for them, and if it's in the cache then an internet connection isn't required.